cypress/e2e/widgets/openshift-rhel.cy.ts

```diff
@@ -0,0 +1,44 @@
+describe('Red Hat OpenShift Widget', () => {
+  const removeWidget = (widgetId: string) => {
+    // we're trying to use IDs instead of selectors, thus "within"
+    cy.get(`[data-ouia-component-id="${widgetId}"]`).within(() => {
+      cy.get('[aria-label="widget actions menu toggle"]').click();
+    });
+    cy.get('[data-ouia-component-id="remove-widget"]')
+      .click()
+      .wait('@patchLayout');
+    cy.get(`[data-ouia-component-id="${widgetId}]`).should('not.exist');
+  };
+
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/');
+
+    cy.intercept(
+      'GET',
+      '**/api/chrome-service/v1/dashboard-templates?dashboard=landingPage'
+    ).as('resetLayout');
+
+    cy.intercept('PATCH', '**/api/chrome-service/v1/dashboard-templates/*').as(
+      'patchLayout'
+    );
+
```

Comment:
Missing a reset call
```suggestion
    cy.resetToDefaultLayout();
```